frontend/i18n-check.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python3
+import json
+import subprocess
+from pathlib import Path
+
+translation_path = Path(__file__).parent / "locales"
+translation_files = translation_path.glob("*.json")
+
+
+def check_empty_value(dictionary: dict) -> bool:
+    for (key, value) in dictionary.items():
+        if value == "":
+            print(f"Translation for '{key}' is missing.")
+            return True
+        if isinstance(value, dict) and check_empty_value(value):
+            return True
+    return False
+
+
+# First check for missing translation keys
+print("Checking for missing translation keys...")
+results = subprocess.run(
+    "npx vue-i18n-extract report --vueFiles './{components,pages,layouts,composables}/**/*.?(ts|vue)' --languageFiles './locales/*.?(json|yml|yaml|js)'",
```

Comment:
Same problem as in package.json:

```suggestion
    'npx vue-i18n-extract report --vueFiles "./{components,pages,layouts,composables}/**/*.{ts,vue}" --languageFiles "./locales/**/*.{json,yml,yaml,js}"',
```